couchdbkit/client.py

```diff
@@ -530,7 +530,7 @@ def get_rev(self, docid):
             if e.response.status_code == 404:
                 raise ResourceNotFound
             raise
-        return response.headers['ETag'].strip('"')
+        return response.headers['ETag'].strip('"').strip('W/"')
```

Comment:
Where is the `W/"` coming from?

Reply:
Never figured out exactly where.  I assume that the cloudant api sends a slightly different request.